sycl/doc/Tests.md

```diff
@@ -19,8 +19,8 @@ working yet are in the issue_related directory.
 All the tests in the simple_tests directory are small unit tests with the
```

Comment:
Since the testing has been significantly changed so i expect that there is still quite a few things to change in this file,

Reply:
I have updated most of them.